source/_integrations/ads.markdown

```diff
@@ -27,6 +27,8 @@ The ADS (automation device specification) describes a device-independent and fie
 
 There is currently support for the following device types within Home Assistant:
 
+- [Configuration](#configuration)
+- [Action](#action)
```

Comment:
This doesn't belong on the list

Reply:
Its automatically added to the list when saving.

Reply:
fixed